replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicationSiteImpl.java

```diff
@@ -29,9 +29,13 @@
 
   private static final String URL_KEY = "url";
 
+  private static final String IS_DISABLED_LOCAL_KEY = "is-disabled-local";
+
   /** 1 - initial version. */
   public static final int CURRENT_VERSION = 1;
```

Comment:
❗️ I think we should bump the version here and default the flag to false when retrieving a version 1